### DISCUSSION TITLE
Change the /silence command

### DISCUSSION THREAD

**Discussion by Coadon** [EssentialsX/Essentials#4103] [Ideas and Feature Suggestions] [answered]
It will be pretty helpful to add the /silence and /unsilence command, it silence the chat and no one can see other player's chat messages, nor can the chat be sent. The console or some players with special permission may still be able to see the silenced chat and send un-silenced chat.

EDIT: I just realized that the /silence command is an alias of the command /mute, so instead of add a command, maybe change this command, or something else as long as it makes sense.

**Comment by JRoy** [comment, marked as answer]
This is not in the scope of EssentialsChat
